.travis.yml

```diff
@@ -0,0 +1,57 @@
+language: android
+jdk: oraclejdk8
+sudo: true
+env:
+  matrix:
+  - ANDROID_TARGET=android-21 ANDROID_ABI=armeabi-v7a
```

Comment:
The matrix is to specify combinations of runtime/environment settings to build in parallel. Here there's only one combination specified, so we're not really leveraging it. We could use a matrix to do something like run tests on android-17, android-21, and android-25, in which case each Travis build would have three sub-builds for each platform.